.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 jdk:
   - oraclejdk8
 addons:
-  sauce_connect: true
+  sauce_connect:
+    username: alloyui
```

Comment:
Here is what Travis specify for this behaviour: https://docs.travis-ci.com/user/sauce-connect/